java-sdk-spring-boot-auto-config/src/main/java/com/yoti/api/spring/YotiClientProperties.java

```diff
@@ -0,0 +1,84 @@
+package com.yoti.api.spring;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * Allows properties to the Yoti client to be supplied via spring properties (e.g. YAML or properties file).
+ */
+@ConfigurationProperties(prefix = "com.yoti.client")
+public class YotiClientProperties {
+
+    /**
+     * The SDK client id provided by Yoti dashboard.
+     */
+    private String clientSdkId;
```

Comment:
should SDK be all uppercase, given it is an acronym? It is not a rhetorical question.